pkg/fatfs/Makefile

```diff
@@ -1,9 +1,11 @@
 PKG_NAME=fatfs
 PKG_URL=https://github.com/RIOT-OS/FatFS
```

Comment:
From your description I expected the upstream URL to be changed. Why is this not the case?

Reply:
There is no upstream repo, only a .zip file and .zip files with .patch files

Reply:
You don't have to use Git for packages. Why not use [HTTPS](https://github.com/RIOT-OS/RIOT/blob/master/pkg/Makefile.http) for it?

Reply:
The upstream site doesn't seem to be very reliable - it is down right now.

Reply:
Should I be able to apply the above as a patch?

Reply:
You can use this as an alternative to `pkg/fatfs/Makefile`, it will then download the `.zip` file from upstream (and not apply any patches, as that needs to be done manually in the `Makefile` when not using a git pkg…)

Reply:
I'm not sure this is worth the hazzle, if it works as is

Reply:
> You can use this as an alternative to `pkg/fatfs/Makefile`, it will then download the `.zip` file from upstream (and not apply any patches, as that needs to be done manually in the `Makefile` when not using a git pkg…)

Erm you should be able to patch using the `patch` command. But if you think having a git fork is easier, then I won't block it. Very weird that those guys don't use any VCS.

Reply:
> Erm you should be able to patch using the `patch` command. 

Yes, adding an additional `patch:` target to the `Makefile` as played out in the example `Makefile.http`.
But since fetching the file is unreliable to begin with, I didn't implement this.